pay-api/src/pay_api/services/eft_short_name_summaries.py

```diff
@@ -118,7 +129,10 @@ def get_search_query(cls, search_criteria: EFTShortnamesSearch):
         ).outerjoin(
             last_payment_subquery,
             and_(last_payment_subquery.c.short_name_id == EFTShortnameModel.id, last_payment_subquery.c.rn == 1)
-        ))
+        )).outerjoin(
+            refund_shortname_subquery,
+            refund_shortname_subquery.c.short_name_id == EFTShortnameModel.id
```

Comment:
Can we have multiple pending short name refunds? It would duplicate rows in that case?

Reply:
it will, I added distinct to avoid duplicate rows

Reply:
where is the distinct?

Reply:
just pushed